storage/gc_queue.go

```diff
@@ -328,18 +328,18 @@ func (gcq *gcQueue) lookupGCPolicy(rng *Replica) (config.GCPolicy, error) {
 		return config.GCPolicy{}, util.Errorf("gossiped info is not a prefix configuration map: %+v", info)
 	}
 
-	// Verify that the range doesn't cross over the zone config prefix.
-	// This could be the case if the zone config is new and the range
+	// Verify that the replica doesn't cross over the zone config prefix.
```

Comment:
This comment should still refer to the range - boundaries and splits (and more generally, anything accessed through `repl.Desc()`) are range-level properties, not replica-level.